src/style.css

```
@@ -0,0 +1,189 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #fff;
+  font-size: 100px;
+}
+
+.container {
+  width: 100vw;
+}
+
+.block_shell {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+}
+
+.block_1 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(255, 0, 0);
+}
+
+.block_2 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(204, 0, 0);
+}
+
+.block_3 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(153, 0, 0);
+}
+
+.block_4 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(102, 0, 0);
+}
+
+.block_5 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(51, 0, 0);
+}
+
+.block_6 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(0, 0, 0);
+}
+
+@media screen and (max-width: 600px) {
+  .block_shell {
+    flex-wrap: wrap;
+  }
+
+  .block {
+    flex-basis: 300px;
+    flex-grow: 1;
+  }
+}
+
+@media screen and (min-width: 600px) and (max-width: 900px) {
+  .block_shell {
+    flex-wrap: wrap;
+  }
+
+  .block {
+    flex-basis: 300px;
+    flex-grow: 1;
+  }
+
+  .block:first-of-type {
+    order: 1;
+  }
+
+  .block:nth-of-type(2) {
+    order: 2;
+  }
+
+  .block:nth-of-type(3) {
+    order: 4;
+  }
+
+  .block:nth-of-type(4) {
+    order: 3;
+  }
+
+  .block:nth-of-type(5) {
+    order: 5;
+  }
+
+  .block:nth-of-type(6) {
+    order: 6;
+  }
+}
+
+@media screen and (min-width: 900px) and (max-width: 1200px) {
```

```suggestion
@media screen and (min-width: 900px) {
```
Нижний медиа запрос перекроет этот, когда наступит его ширина экрана